packages/ui/pages/workspace/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { AppLayout } from 'components/AppLayout';
+import Link from 'next/link';
+import {
+  MultipleProgram,
+  WorkspaceHeader,
+} from 'components/Pages/Workspace';
+
+function Workspace() {
+  const [isFirstTimeVisitor] = useState(true);
+
+  return (
+    <AppLayout>
+      <div className="bg-gray-300 h-screen flex flex-col">
+        <WorkspaceHeader
+          description={
+            isFirstTimeVisitor
+              ? 'Our goal today is to get you one step ahead in your gene therapy treatment roadmap'
+              : 'Let us see what you got'
+          }
+        />
+        {isFirstTimeVisitor ? (
+          <div className="px-4 flex justify-center mt-8">
+            <div className="bg-white w-full py-20 flex flex-col justify-center items-center">
+              <div>
+                <img
+                  src="/OPenGT Illustrations-25 1@2x.png"
+                  width={400}
```

Comment:
@pipesort123TP need to downcase the name without any spaces any referenced static files.